01-HTML-CSS-Basics/1. HTML/Punto 1/iv. expenses.html

```diff
@@ -0,0 +1,17 @@
+<head>
+	<p>Expenses</p>
+	<table>
+		<tr>
+			<td>Party</td>
```

Comment:
You can also use th (better placed inside a thead tag) to specify cells that have a heading purpose.